docs/reference/release-notes/6.7.asciidoc

```diff
@@ -65,6 +65,20 @@ SQL::
 
 Also see <<breaking-changes-6.7,Breaking changes in 6.7>>.
 
+[float]
+=== Known issues
+
+A new `docker` `build_type` value was introduced in 6.7.0 for Docker images,
+which caused two problems:
+
+* If you perform a rolling upgrade to 6.7.0 on nodes that are running the
+official Docker container, they cannot rejoin the cluster. Upgrade to 6.7.1
+instead or perform a <<restart-upgrade,full cluster restart upgrade>>. (issue: {issue}/40784[#40784])
+* If you have a 6.6 or earlier Java High Level REST client, it cannot understand
+the new `build_type` value. For example, it cannot read main responses on `/`
+from a cluster that's running 6.7.0. Upgrade your cluster to 6.7.1 instead.
```

Comment:
Unfortunately this issue is not addressed in 6.7.1; the only workaround here is to upgrade the client to at least 6.7.0.

Reply:
Thanks, I've fixed that work-around.